README.md

```diff
@@ -1044,12 +1053,12 @@ __This library is faster__ for encoding and decoding CBOR Web Token (CWT).  Howe
 
 __This library uses less memory__ for encoding and decoding CBOR Web Token (CWT) using test data from RFC 8392 A.1.
 
-|  | fxamacker/cbor 2.2 | ugorji/go 1.1.7 |
+|  | fxamacker/cbor 2.3 | ugorji/go 1.2.6 |
 | :--- | :--- | :--- | 
-| Encode CWT | 176 bytes/op &nbsp;&nbsp;&nbsp; 2 allocs/op | 1424 bytes/op &nbsp;&nbsp;&nbsp; 4 allocs/op |
-| Decode CWT | 176 bytes/op &nbsp;&nbsp;&nbsp; 6 allocs/op | &nbsp; 568 bytes/op &nbsp;&nbsp;&nbsp; 6 allocs/op |
+| Encode CWT | 0.18 kB/op &nbsp;&nbsp;&nbsp; 2 allocs/op | 1.35 kB/op &nbsp;&nbsp;&nbsp; 4 allocs/op |
+| Decode CWT | 160 bytes/op &nbsp;&nbsp;&nbsp; 6 allocs/op | &nbsp; 744 bytes/op &nbsp;&nbsp;&nbsp; 6 allocs/op |
```

Comment:
We can remove some extra white spaces.  For example, the white space in front of `744 bytes`.